mnefun/_sss.py

```diff
@@ -696,7 +707,7 @@ def _mf_autobad(raw, p, skip_start, skip_stop):
     return bads
 
 
-def _get_fit_data(raw, p=None, prefix='    '):
+def _get_fit_data(raw, p=None, prefix='    ', return_t_window=False):
```

Comment:
rather than add a parameter, this func is private so we can just always return it and update all uses of `_get_fit_data` in `mnefun`